environment_v2_linux_cpuonly.yml

```diff
@@ -0,0 +1,56 @@
+name: gtsfm-v2
```

Comment:
Just curious, why the new env file? Should we just update v1 @ayushbaid ?

Reply:
Removed now.

Reply:
Marking as resolved, thanks!